omnigibson/prims/cloth_prim.py

```diff
@@ -83,11 +83,14 @@ def _post_load(self):
         if "mass" in self._load_config and self._load_config["mass"] is not None:
             self.mass = self._load_config["mass"]
 
+        # Internal reference to the monolithic cloth system
+        self._cloth_system = get_system("cloth")
```

Comment:
this is global right? should we just make this a class property / class attribute?

Reply:
done